scripts/build/config.mjs

```diff
@@ -48,6 +48,8 @@ const parsers = [
       'require("perf_hooks")': "{}",
       'require("inspector")': "{}",
       "_fs.realpathSync.native": "_fs.realpathSync && _fs.realpathSync.native",
+      // Remove useless `ts.sys`
+      "ts.sys = ": "ts.sys = undefined && ",
```

Comment:
Can you tell me where `ts.sys =` is written? I couldn't find it from non-minified `parser-typescript.js`...

Reply:
`node_modules/typescript/lib/typescript.js`

Reply:
`non-minified parser-typescript.js` use `ts2.sys`

Reply:
There are many useless stuff, and it can't be tree-shaked, I've watched https://github.com/microsoft/TypeScript/issues/27891 for years.. still no progress.

Reply:
Thank you. I got it.